oracle-plugin/src/e2e-test/features/oracle/Oracle.feature

```diff
@@ -66,3 +66,4 @@ Feature: Oracle - Verify Oracle source data transfer
     Then Verify the pipeline status is "Succeeded"
     Then Get count of no of records transferred to target Oracle Table
     Then Validate records transferred to target table is equal to number of records from source table
```

Comment:
This is not required to be performed explicitly we can do that internally as well as part of the next step.

Reply:
Removed